### PR TITLE
Added fellowship 2024 award

### PR DESCRIPTION
**Pull request by matteopilz**

### CODE REVIEW

**Comment by timosachsenberg** [comment]
so we don't host one?

File: content/en/news/fellowship2024.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  </div>
+</div> 
+
+<br>
+
+# Supporting Society
+   Females in Mass Spectrometry (FeMS) is a community-led initiative to create a network of
+   support for women in the field of mass spectrometry. Our goal is to bring together, encourage,
+   empower and facilitate viability of women in mass spectrometry - and to create connectivity
+   between all members of the mass spectrometry com![OpenMS.svg](..%2F..%2F..%2F..%2F..%2FDownloads%2FOpenMS.svg)munity.
```
Comment (tjeerdijk) [inline content/en/news/fellowship2024.md:105]: PR looks good except for this line. Can you check?

**Comment by matteopilz** [comment]
The cropping was also from a change in the style sheet css file. I have changed the way the logos are displayed.

File: content/en/news/fellowship2024.md (hunk 1 above)
Comment (tjeerdijk) [reply content/en/news/fellowship2024.md]: Thx

**Review by tjeerdijk — APPROVED** [review]
Looks good now

**Pull request merged**